docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -518,6 +518,57 @@ useIntegerType (``boolean``)
             }
         }
 
+.. _generate-openapi-setting-onErrorStatusConflict:
+
+onErrorStatusConflict (``String``)
+    Specifies how to resolve multiple error responses that share a same HTTP status code.
+    This behavior can be customized using the following values for the ``onErrorStatusConflict`` setting:
+
+    ``oneOf``
+        Use OpenAPI's ``oneOf`` keyword to combine error responses with same HTTP status code.
+
+
+    ``properties``
+        Use ``properties`` field of OpenAPI schema object to combine error responses with same HTTP status code.
+
+    .. note::
+            ``oneOf`` keyword is not supported by Amazon API Gateway.
+
+    Both options generate a single combined response object called "UnionError XXX Response" in the
+    OpenAPI model output, where "XXX" is the status code shared by multiple errors. Both options drop
+    the ``@required`` trait from all members of conflicting error structures, making them optional.
+
+    ``oneOf`` option wraps schemas for contents of conflicting errors responses schemas into a synthetic union schema
+    using OpenAPI's ``oneOf`` keyword.
+
+    ``properties`` option pre-processes the Smithy model before OpenAPI conversion takes place, during which
+    conflicting error structure shapes are combined into one union error shape with all members from each and every
+    conflicting error.
```

Comment:
About this one, I think that the implementation details can be left off and rather just explain what it does. Instead something like:

" all the properties from the conflicting structures are combined in a single structure ..."

Reply:
Modified as suggested by comment.